src/Task/ApiGen/ApiGen.php

```diff
@@ -48,6 +59,31 @@ public function __construct($pathToApiGen = null)
     }
 
     /**
+     * Pass methods parameters as arguments to executable. Argument values
+     * are automatically escaped.
+     *
+     * @param string|string[] $args
+     *
+     * @return $this
+     */
+    public function args($args)
+    {
+        if (!is_array($args)) {
+            $args = func_get_args();
+        }
+        $args = array_map(function ($arg) {
+            if (preg_match('/^\w+$/', trim($arg)) === 1) {
```

Comment:
Pulling the operation out of the args seems error-prone to me, or at least inconsistent. Provide another method `operation()` to set the op.